app/services/rbac.rb

```diff
@@ -35,6 +37,17 @@ def load_user_permissions(identity)
       end
     end
 
+    # Scoped for inventory groups
+    def definition_value(definition, selector)
```

Comment:
I would also parametrize the `operation` which would make the method universal and not just scoped for inventory groups.